edx-solutions/xblock-drag-and-drop-v2
# chore: updates translation files and better transifex mapping

improves translation for nl, ja, de, ko, it, pl, tr, es_419, hi and zh_CN languages

**Testing instructions:**
- Create a new course and add a new DnDv2 problem
- Edit the DnDv2 problem to use the Assessment mode
- Change the user's language going to /update_lang/ (lms), choose one supported by this xblock (check "LANGUAGES" inside this repository settings file)
- Go back to the problem and confirm the problem is being translated correctly (`nl` and `ja` were not supported before, so they should be available now for example)
